EIPS/eip-7831.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: 7831
```

Comment:
```suggestion
eip: 7833
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.